### ISSUE TITLE
layout template within theme project not correctly deployed

### ISSUE THREAD

**Issue by dermoritz** [liferay/generator-liferay-theme#18]
We are trying to create a layout template within our theme project generated by "Liferay Theme Generator". We followed [the doc](https://dev.liferay.com/de/develop/tutorials/-/knowledge_base/7-0/creating-layout-templates-with-the-themes-generator-0) and ran the generator within our theme-project folder.
The generation went through without error and yielded src\layouttpl\custom\1_Column_747-layouttpl with the appropriate files in there.
The problem is that this template is not available after gulp deploy while the theme is. I think this is caused by a missing "liferay-layout-templates.xml" in WEB-INF that should be generated, shouldn't it?

**Comment by Robert-Frampton** [comment]
Hey @dermoritz,

It looks like the generator doesn't create the `<layout-templates>` section in the theme's `liferay-look-and-feel.xml`. Take a look at this theme in portal as an example https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme-porygon/frontend-theme-porygon/src/WEB-INF/liferay-look-and-feel.xml#L16-L27.

We'll need to add this functionality to the generator, but in the meantime you can add that config directly to your `liferay-look-and-feel.xml` file.

Also take note of a reported issue with theme layout templates in Portal https://issues.liferay.com/browse/LPS-71596. It's fixed now, so just depends on what fixpack you're on.

**Comment by mwilliams2014** [comment]
Thanks for letting us know @dermoritz ! We'll update the doc to include this step. You can track the progress here: https://issues.liferay.com/browse/LRDOCS-3909

**Comment by blzaugg** [comment]
@mwilliams2014, for the docs, can we highlight, that this is a workaround, until it's fixed in the generator?

**Comment by mwilliams2014** [comment]
Yep. Will do @blzaugg.

**Comment by dermoritz** [comment]
Thanks for fast response. Meanwhile i added a separate liferay-layout-templates.xml (beside the existing liferay-look-and-fell.xml) this works fine. I took this road because i am new to liferay and saw this file beeing generated by the generator if i create an layout template in an empty folder. I only had to adapt the paths to the files (/layouttpl/custom/...)
If it is ok to have an separate file (for me it works fine) than it is probably much easier to implement for you?!
